### DISCUSSION TITLE
wogg.link访问要被检测是否真人

### DISCUSSION THREAD

**Discussion by paulyu666** [IrineSistiana/mosdns#783] [General]
我用tvbox有个经常看的源叫“玩偶哥哥”，突然现在访问要经过cloudflare的安全认证，导致我的tvbox无法访问，在白名单内加了wogg的关键词，还是不行，我用手机网络时没问题的，哪个大佬帮忙看下，谢谢
<img width="618" alt="image" src="https://github.com/IrineSistiana/mosdns/assets/64824531/b4397eaa-cca1-406b-910b-26fd5baf3e36">
<img width="767" alt="image" src="https://github.com/IrineSistiana/mosdns/assets/64824531/3739f30d-6937-4731-9934-ab026fd03a5a">

**Comment by yuukh** [comment]
和dns无关，网站的问题或你出口ip不干净
